tests/unit/conftest.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/python3
+
+# conftest.py
+# Date:  18/11/2020
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com 
+
+""" """
+import json
+import pathlib
+import shutil
+
+import pandas as pd
+import pytest
+import rdflib
+
+import lam4vb3.lam_utils
+from lam4vb3 import LAM_PROPERTIES_WS_NAME, LAM_CLASSES_WS_NAME, LAM_PROPERTY_CLASSIFICATION_WS_NAME, \
+    LAM_CLASS_CLASSIFICATION_WS_NAME, CELEX_CLASSES_WS_NAME, CELEX_CLASS_CLASSIFICATION_WS_NAME, PREFIX_WS_NAME
+from lam4vb3.builder.celex_classes_builder import make_celex_classes_worksheet
+from lam4vb3.builder.lam_classes_builder import make_lam_classes_worksheet
+from lam4vb3.builder.property_builder import make_property_worksheet
+from lam4vb3.excel2rdf import transform_celex_classes, transform_properties, transform_classes
+from lam4vb3.lam_utils import read_excel_worksheet
+from tests import OUTPUT_FOLDER, THIS_PROJECT
+
+SHACL_SHAPES_2021 = pathlib.Path(__file__).parent.parent.parent / "models" / "lam-skos-ap-2021.ttl"
+TESTBED_EXCEL_2021_08 = pathlib.Path(__file__).parent.parent / "test_data" / "LAM_metadata_20210413_testbed.xlsx"
+TEMP_OUTPUT_FOLDER = pathlib.Path(__file__).parent.parent / "output"
+
+# file fixtures
+
+@pytest.fixture(scope="session")
+def shacl_shapes() -> rdflib.Graph:
+    result_graph = rdflib.Graph()
+    result_graph.parse(source=str(SHACL_SHAPES_2021))
+    return result_graph
+
+
+# example queries
+
+@pytest.fixture(scope="session")
+def lam_property_author_query():
+    path_to_query_file = pathlib.Path(__file__).parent.parent / "queries" / "lam_property_author.rq"
+    return path_to_query_file.read_text()
+
+
+@pytest.fixture(scope="session")
+def lam_property_author_example_query():
+    path_to_query_file = pathlib.Path(__file__).parent.parent / "queries" / "lam_property_author_example.rq"
```

Comment:
Set this PATH (pathlib.Path(__file__).parent.parent / "queries") as constant and reuse it.